multisig/proposals/ethereum/vip-converter/Addresses.ts

```diff
@@ -0,0 +1,51 @@
+export const Assets = [
+  "0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E", // CRV_USD
+  "0xD533a949740bb3306d119CC777fa900bA034cd52", // CRV
+  "0x6B175474E89094C44Da98b954EedeAC495271d0F", // DAI
+  "0x853d955aCEf822Db058eb8505911ED77F175b99e", // FRAX
+  "0xA663B02CF0a4b149d2aD41910CB81e23e1c41c32", // SFRAX
+  "0x6ee2b5E19ECBa773a352E5B21415Dc419A700d1d", // weETH-26DEC2024
+  "0x0000000000085d4780B73119b644AE5ecd22b376", // TUSD
+  "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // USDC
+  "0xdAC17F958D2ee523a2206206994597C13D831ec7", // USDT
+  "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599", // WBTC
+  "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2", // WETH
+  "0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee", // W_E_ETH
+  "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0", // WST_ETH
```

Comment:
I would add the addresses of rsETH and sfrxETH, that will be added soon:
* rsETH: 0xA1290d69c65A6Fe4DF752f95823fae25cB99e5A7 (https://github.com/VenusProtocol/vips/pull/304)
* sfrxETH: 0xac3E018457B222d93114458476f3E3416Abbe38F (https://github.com/VenusProtocol/vips/pull/302)

Reply:
https://github.com/VenusProtocol/vips/pull/303/commits/d04f88d5fa97464854791a6e5ec1d52425e6d0ed